### PR TITLE
Sass 3529 Student Loan iterations

### PR DESCRIPTION
**Pull request by satheeshgottemukkula**
### Description
Please include a brief summary of the change you've made and the issue that it fixes.

SASS-3529 Student Loan iterations 
https://jira.tools.tax.service.gov.uk/browse/SASS-3529

##### Before Reviewing
- [ ]  Have you pulled the branch down?
- [ ]  Have you assigned yourself to the PR?
- [ ]  Have you moved the task to “in review” on JIRA?
- [ ]  Have you checked to ensure all dependencies are up to date?
- [ ]  Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing
- [ ]  Have you run the tests?
- [ ]  Have you run the journey tests?
- [ ]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [ ]  Have you checked for merge conflicts or any changes in the current main that may affect the current pull request? i.e. does it need another rebase?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [x]  Have you run the journey tests? (where applicable)
- [ ]  Have you addressed warnings where appropriate?
- [x]  Have you rebased against the current version of main?
- [x]  Have you checked code coverage isn’t lower than previously?

##### After PRs been raised
- [ ]  Have you checked the PR Builder passes?

### CODE REVIEW

File: app/views/studentLoans/StudentLoansQuestionView.scala.html [hunk 1]
```diff
@@ -72,7 +72,7 @@
                     content = Text(messages("studentLoansQuestion.checkbox.2")),
                     value = "pgl",
                     hint = Some(Hint(
```
Comment (georgievh) [inline app/views/studentLoans/StudentLoansQuestionView.scala.html:74]: If no Hint is needed you can remove the parameter (I think). I think the default value is None, so you don't need to pass anything.

File: conf/app.routes [hunk 2]
```diff
@@ -319,11 +319,11 @@ POST        /:taxYear/employment/expenses/start-employment-expenses
...
 
-GET         /:taxYear/student-loans/postgraduate-repayment-amount                    controllers.studentLoans.PglAmountController.show(taxYear: Int, employmentId: String)
-POST        /:taxYear/student-loans/postgraduate-repayment-amount                    controllers.studentLoans.PglAmountController.submit(taxYear: Int, employmentId: String)
+GET         /:taxYear/student-loans/postgraduate-loan-repayment-amount               controllers.studentLoans.PglAmountController.show(taxYear: Int, employmentId: String)
+POST        /:taxYear/student-loans/postgraduate-loan-repayment-amount               controllers.studentLoans.PglAmountController.submit(taxYear: Int, employmentId: String)
 
-GET         /:taxYear/student-loans/undergraduate-repayment-amount                   controllers.studentLoans.UglAmountController.show(taxYear: Int, employmentId: String)
-POST        /:taxYear/student-loans/undergraduate-repayment-amount                   controllers.studentLoans.UglAmountController.submit(taxYear: Int, employmentId: String)
```
Comment (georgievh) [inline conf/app.routes:326]: Those changes will possibly fail the performance tests. Please ensure those are updated before merging

File: conf/messages [hunk 3]
```diff
@@ -1337,39 +1337,40 @@ expenses.removeExpenses.button = Remove expenses
 
 #####################    Student Loans Question messages    #####################
 
-studentLoansQuestion.title.individual = Did you repay any student loan?
-studentLoansQuestion.title.agent = Did your client repay any student loan?
-studentLoansQuestion.heading.individual = Did you repay any student loan while employed by {0}?
-studentLoansQuestion.heading.agent = Did your client repay any student loan while employed by {0}?
+studentLoansQuestion.title.individual = Did you repay any student loans?
```
Comment (georgievh) [inline conf/messages:1340]: Those changes will break Journey tests and potentially smoke tests. Have those been updated?

File: conf/messages [hunk 4]
```diff
@@ -1337,39 +1337,40 @@ expenses.removeExpenses.button = Remove expenses
...
 
-studentLoans.undergraduateLoanAmount.title.individual = How much undergraduate loan did you repay?
-studentLoans.undergraduateLoanAmount.title.agent = How much undergraduate loan did your client repay?
-studentLoans.undergraduateLoanAmount.heading.individual = How much undergraduate loan did you repay while employed by {0}?
-studentLoans.undergraduateLoanAmount.heading.agent = How much undergraduate loan did your client repay while employed by {0}?
-studentLoans.undergraduateLoanAmount.label = How much undergraduate loan did you repay with this employer
-studentLoans.undergraduateLoanAmount.error.noEntry.individual = Enter the amount of undergraduate loan you repaid while employed by {0}
-studentLoans.undergraduateLoanAmount.error.noEntry.agent = Enter the amount of undergraduate loan your client repaid while employed by {0}
-studentLoans.undergraduateLoanAmount.error.invalidFormat = Enter the amount of undergraduate loan in the correct format
+studentLoans.undergraduateLoanAmount.title.individual = How much student loan did you repay?
```
Comment (georgievh) [inline conf/messages:1357]: Same as above. We need to make sure the other tests will pass.

**Review by georgievh — CHANGES_REQUESTED** [review]

File: conf/app.routes (hunk 2 above)
Comment (satheeshgottemukkula) [reply conf/app.routes:326]: Performance tests and journey tests are updated

File: conf/messages (hunk 3 above)
Comment (satheeshgottemukkula) [reply conf/messages:1340]: Journey tests updated and will check for smoke tests

**Comment by jhkldev** [comment]
test this please

**Review by jhkldev — APPROVED** [review]

**Pull request merged**